Assignment3-Loops-Siffrinn/src/assignment3/loops/siffrinn/Assignment3LoopsSiffrinn.java

```diff
@@ -0,0 +1,38 @@
+package assignment3.loops.siffrinn;
+
+import java.util.Scanner;   //needed for the Scanner class
+
+//This program will show the reverse of a word
+
+public class Assignment3LoopsSiffrinn {
+  
+    public static void main(String[] args) {
+ 
+        String origWord;    //original word
+        
+       Scanner keyboard = new Scanner(System.in); //initialize the scanner
+        
+        //Let's user choose a word and stores it in the variable origWord
+        System.out.println("Pick a word with at least 5 letters.\n");
+        origWord = keyboard.nextLine();
+        
+        //line space used for better output
+         System.out.print("\nThe reverse of your word is ");
+         
+        //to reverse the word
+        for(int i = 1; i <origWord.length()+1; i++){
+         
+        //to print the reversed word
+        System.out.print(origWord.charAt(origWord.length()-i));
+          
```

Comment:
@Arevyn24 Whew, after struggling with this because I was thinking too big and over complicating this, I took a break. When I came back, I started over.  I must be honest... I wasn't sure this was going to work. It seemed right in my head but it was so simple, I figured it must be wrong. One thing our instructor Paul has taught me is.... Run it! Write code and run it. See what happens. It's free.